routing/route/route.go

```diff
@@ -175,3 +177,19 @@ func (r *Route) ToSphinxPath() (*sphinx.PaymentPath, error) {
 
 	return &path, nil
 }
+
+// String returns a human readable representation of the route.
+func (r *Route) String() string {
+	var b strings.Builder
+
+	b.WriteString(strconv.FormatUint(r.Hops[0].ChannelID, 10))
```

Comment:
do we want to put this in a conditional, so that it can't panic if the route is empty?

Reply:
good point, fixed